### ISSUE TITLE
Syntax highlight triple backticked (pre) sections

### ISSUE THREAD

**Issue by mcandre** [plasticboy/vim-markdown#175]
Could we have vim-markdown highlight triple-backticked (` ` `) sections as PRE-formatted code blocks?

**Comment by lazyfrosch** [comment]
You mean the actual syntax highlighting of the code? That might be hard

**Comment by mcandre** [comment]
No, haha. Just highlight all text inside the backticks as one uniform color.

**Comment by lazyfrosch** [comment]
Can you give a code example where it does not work?

**Comment by mcandre** [comment]
Sure!

None of the triple-backticked code sections in https://raw.githubusercontent.com/mcandre/cheatsheets/master/vim.md are highlighted.

I'm using Benokai for my color theme, in case that matters.

**Comment by cirosantilli** [comment]
@mcandre I don't reproduce: to me they appear with the same color / syntax ID as inline code:

    out `inline` in

Does your inline code highlight?

**Comment by mcandre** [comment]
No, my inline code does not highlight either. I'll bring Benokai into the conversation in case it's to blame.

**Comment by cirosantilli** [comment]
It looks like it's on Benokai.

More precisely, the syntax element is called `mkdCode`, and the line:

    HtmlHiLink mkdCode          String

says that it looks like `String`, which is a standard Vim syntax element. So it this is the case, C-style strings like `"adsf"` in C code should not be highlighted either.

You could have found about `mkdCode` with:

        set laststatus=2
        set statusline=%{synIDattr(synID(line('.'),col('.'),1),'name')}

and hovering the cursor over the element.

**Comment by citizenmatt** [comment]
See #93
